recipes/libcurl/all/conanfile.py

```diff
@@ -536,10 +536,8 @@ def package(self):
 
     def package_info(self):
         self.cpp_info.set_property("cmake_file_name", "CURL")
-        self.cpp_info.set_property("cmake_target_name", "CURL")
+        self.cpp_info.set_property("cmake_target_name", "CURL::libcurl")
```

Comment:
`cmake_find_mode` should be set to `both` since there is a builtin FindCURL.cmake in CMake.